cunumeric/thunk.py

```diff
@@ -52,7 +57,7 @@ def size(self):
     # Abstract methods
 
     @abstractproperty
-    def storage(self):
+    def storage(self) -> Any:
```

Comment:
I think based on the definition of `DeferredArray.storage()` this is guaranteed to return `Union[Future, Tuple[Region, FieldID]]` or something like that.

Reply:
yup thanks https://github.com/nv-legate/cunumeric/pull/438/commits/dde3fabcc7fd0c1e44b1535d93c24b0bc1144d92